ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/operations/SModOperation.java

```diff
@@ -33,11 +38,28 @@ public Gas cost(final MessageFrame frame) {
 
   @Override
   public void execute(final MessageFrame frame) {
```

Comment:
Test coverage ?

Reply:
?

Reply:
Just want to make sure the method is not broken. Do we have reference tests for that ?

Reply:
The smod operation is well covered by reference tests. Did you have something else in mind?

Reply:
No that's fine. Thanks.

Reply:
\m/